### ISSUE TITLE
Right-clicking on some packages in the "Package Management" window pop-ups a very small window with scroll bars

### ISSUE THREAD

**Issue by kongondo** [modxbot/migrate#1432]
kongondo created Redmine issue ID 1432
1. Right-clicking on some packages in the "Package Management" window pop-ups a very small window with scroll bars (see screens). I am referring to the window with "install" "update"...etc package Screens url: http://i414.photobucket.com/albums/pp228/kongondo/bug_right_click_pckg_mgnt_revo_svn_.png
   http://i414.photobucket.com/albums/pp228/kongondo/bug_right_click_pckg_mgnt_revo_s-1.png
   http://i414.photobucket.com/albums/pp228/kongondo/bug_right_click_pckg_mgnt_revo_s-2.png Thanks
   /kongondo

**Comment by splittingred** [comment]
splittingred submitted:

No matter how much I try, I can't seem to duplicate this issue on the Mac.

**Comment by splittingred** [comment]
splittingred submitted:

Nor can our PC testers. Is this happening consistently? Are you getting any errors in the Browser?

**Comment by kongondo** [comment]
kongondo submitted:

It is happening consistently but not across browsers. In FF 3.5, for some packages, right-clicking twice (not double clicking) seems to fix it. This is currently happening with the 3 bottom-most packages in the grid, in my case those happen to be UltimateParent, Wayfinder and Xternalinks. If I right click the first time, a "small" window pops up. It is positioned above the Package itself. Righ-clicking a second time pops a normal window but below the Package itself...i.e. toggling between small window up, normal window down, etc. For some packages, such as Quip, the clicking twice doesn't help and actually the window that pops-up is even smaller. Except for the smaller window, it is the same case with Personalize, Membercheck and Login. Now for the interesting part. I have noticed that the behaviour is actually related to to where the browsers right scroll bar is! If the scroll bar is right at the top, then the right click pops up the window fine! (With my screen resolution, I can see up to and including Quip) . If I move the browser scroll bar right down, the problems start, i.e. with right click, I get small windows on some packages. I have noticed similar behaviour on Safari 4.0.2. The scroll bar affects the size of the pop up window. However, in Safari (probably because I have more real estate than in FF), not all the packages affected in FF are affected. For instance, SPForm is affected in Safari but not in FF. My conclusion, obviously this has nothing to do with the packages themselves. It has to do with the screen resolution/browser window size. Testing on a 1280X800, this happens. Changing to a 800x600, the problem disappears! Testing in 1600x1200, the problem re-appears and affects some other packages includng SPForm in FF 3.5. So, does it have to do with how the script is calculating the viewport? Many thanks. EDIT: Screen size above is 1280x800 (not 1200X800)
